application/actions/auth.go

```diff
@@ -42,6 +42,11 @@ const ReturnToSessionKey = "ReturnTo"
 // http param for token type
 const TokenTypeParam = "token-type"
 
+// environment variable key for the UI's URL
+const UIURLEnv = "UI_URL"
```

Comment:
This could be used in app.og and auth_test.go also.

Reply:
Done